boards/arm/v2m_musca/v2m_musca_defconfig

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2018 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_ARM=y
+CONFIG_SOC_SERIES_MUSCA=y
+CONFIG_SOC_V2M_MUSCA_A=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_RUNTIME_NMI=y
+CONFIG_TRUSTED_EXECUTION_SECURE=y
```

Comment:
My only concern, here is: what if the user wants to use the Musca board as a legacy (no TEE) support?
Why is the TRUSTED_EXECUTION_SECURE set by default? Shouldn't this be better set on the project level? I don't really mind, actually; I am just throwing the idea.

Reply:
well, i think we were using two different defconfigs to determine secure or non-secure.  A non-TEE system would be running secure with SDAU/IDAU configured to fully open, correct?

Reply:
we can remove the TEE from secure, but it has to remain for non-secure because otherwise it is not a sane config.  Then it's up to the application to define whether they want a TEE secure image or a v7m compatible.

Reply:
Ok I see your point.  We just have to punt this to the application.  This means each possible combination of an application (secure, nonsecure, v7m compat) has to be handled individually in that application's config file properly.  This will make reusing current apps troublesome.